Assets/HoloToolkit/Utilities/Tests/Scripts/GestureResponder.cs

```diff
@@ -1,11 +1,31 @@
-﻿using UnityEngine;
-using System.Collections;
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
I believe that any contributions to the HoloToolkit requires you to add this, regardless of employment status. @NeerajW could you clarify this for us?